addons/medical/scripts/Game/ACE_Medical/Components/Damage/SCR_CharacterDamageManagerComponent.c

```diff
@@ -84,12 +86,17 @@ modded class SCR_CharacterDamageManagerComponent : SCR_DamageManagerComponent
 	}
 
 	//------------------------------------------------------------------------------------------------
-	//! Returns true if at least one physical hit zone is injured
-	bool ACE_Medical_CanBeHealed()
+	//! Returns true if at least one physical hit zone is below the provided scaled health. Defaults to 99% health (Any damage to any limb).
+	bool ACE_Medical_CanBeHealed(float maxHealth = -1)
```

Comment:
Default should probably be `0.999` as you wrote in the description above. Also the argument should probably be called `maxHeal` to be consistent with `m_fACE_Medical_MedicalKitMaxHeal`

Reply:
Parameter name has been updated, default has been changed to 0.999 to be more consistent with it's previous usage. Documentation about how to access the medicalkitmaxheal variable has been added.